src/main/java/seedu/address/security/Security.java

```diff
@@ -12,6 +12,17 @@
      */
     void login(String username, String password);
 
+    /***
+     * Logs out
+     */
+    void logout();
+
+    /***
+     * Registers the user
+     * Returns 1 if successful, 2 if user exists, 3 if the fields are incomplete
+     */
+    int register(String username, String password, String email, String phone, String address);
```

Comment:
I went to check it out, enums are constants? How would it be helpful

Reply:
I went to check it out, enums are constants? How would it be helpful

Reply:
https://crunchify.com/why-and-for-what-should-i-use-enum-java-enum-examples/

tl;dr 

> If you use enums instead of integers (or String codes), you increase compile-time checking and avoid errors from passing in invalid constants, and you document which values are legal to use.